packages/button/stories/button.stories.tsx

```diff
@@ -11,6 +11,7 @@ import { FaFacebook, FaTwitter } from "react-icons/fa"
 import { MdBuild, MdCall } from "react-icons/md"
 import { BeatLoader } from "react-spinners"
 import { Button, ButtonGroup, IconButton } from "../src"
+import "focus-visible"
```

Comment:
I don't think this is needed. It has the same issue without this dependency

Reply:
Whoops, didn't mean to include this! Had added it just for testing locally with Storybook. I'll remove that